fdbt-aws/cloudformation/service/eu-west-2/rds.yml

```diff
@@ -70,6 +61,7 @@ Resources:
       Parameters:
         aurora_load_from_s3_role:
           Fn::ImportValue: !Sub ${Stage}:RdsAuroraS3RoleArn
+        max_connections: 90
```

Comment:
How did we come up with `90`? Are we just guessing for the time being?

Reply:
90 is what the production database has by default, so bringing Test env in line with prod